src/components/atoms/Dropdown/Dropdown.scss

```diff
@@ -87,9 +103,10 @@
       display: none;
     }
 
+    // &__Select__value-container--has-value
     &__value-container {
+      //overflow: hidden; // NOTE: makes the static text move up and hide
```

Comment:
remove?

Reply:
I was considering if the comment is enough or remove it all